pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -433,6 +443,30 @@ func (admitter *VMsAdmitter) validateVolumeRequests(vm *v1.VirtualMachine) ([]me
 
 }
 
+func validateRestoreStatus(ar *admissionv1.AdmissionRequest, vm *v1.VirtualMachine) []metav1.StatusCause {
```

Comment:
better refactor this one together with validateSnapshotStatus which are almost similar